test/orm/test_session.py

```diff
@@ -609,6 +609,24 @@ def check(ctx: ORMExecuteState) -> None:
 
         is_true(called)
 
+    def test_get_one_no_result(self):
```

Comment:
**mike bayer** (zzzeek) wrote:

consolidate the test_query -> test_get_one() case to be here where this one is

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4872

Reply:
Tests updated!